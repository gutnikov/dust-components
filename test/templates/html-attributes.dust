<div>
	{@Button className="simple-button" name="button-name" id="button-id" disabled="disabled" text="I'm a button"/}
</div>
