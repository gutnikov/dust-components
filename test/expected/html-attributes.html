<div>
    <button class="button simple-button" name="button-name" id="button-id" disabled="disabled">I'm a button</button>
</div>
